### PR TITLE
Generating changelog automatically from git

### PR DESCRIPTION
**Pull request by kubamahnert**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -142,6 +142,29 @@ namespace :license do
   end
 end
 
+namespace :changelog do
+  desc 'Updates the changelog with commit messages'
+  task :update, [:new_version] do |t, args|
```
Comment (yenkins) [inline Rakefile]: Lint/UnusedBlockArgument: Unused block argument - `t`. If it's necessary, use `_` or `_t` as an argument name to indicate that it won't be used.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/575/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/366/ : FAILURE in 1m 53s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/671/ : SUCCESS in 33s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/576/ : SUCCESS in 39s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/367/ : SUCCESS in 36s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/672/ : SUCCESS in 32s

**Comment by panjan** [comment]
ok to merge

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/232/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/368/ : SUCCESS in 1m 45s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/673/ : SUCCESS in 1m 52s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1131/ : SUCCESS in 47m 15s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/888/ : FAILURE in 37s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/233/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/369/ : FAILURE in 49s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/674/ : SUCCESS in 33s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1132/ : SUCCESS in 47m 14s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/890/ : SUCCESS in 24m 37s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/234/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/371/ : SUCCESS in 2m 07s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/675/ : SUCCESS in 1m 47s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1133/ : SUCCESS in 47m 37s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/891/ : SUCCESS in 27m 07s

**Pull request merged**